shared/actions/settings.tsx

```diff
@@ -775,6 +778,12 @@ function* settingsSaga(): Saga.SagaGenerator<any, any> {
   yield* Saga.chainAction<SettingsGen.LoadProxyDataPayload>(SettingsGen.loadProxyData, loadProxyData)
   yield* Saga.chainAction<SettingsGen.SaveProxyDataPayload>(SettingsGen.saveProxyData, saveProxyData)
 
+  // Runtime Stats
+  yield* Saga.chainAction<SettingsGen.ToggleRuntimeStatsPayload>(
```

Comment:
This should all belong in config. Not across all these various sub parts.

Reply:
Ok, this is on the settings screen so I stuck it here, but can move it.